pyomo/contrib/pyros/separation_problem_methods.py

```diff
@@ -428,6 +428,9 @@ def solver_call_separation(model_data, config, solver, solve_data, is_global):
         # === Process result
         is_violation(model_data, config, solve_data)
 
+        if solve_data.termination_condition == tc.infeasible:
+            print()
```

Comment:
This looks like debugging? (`print()` just prints a blank line)

Reply:
It was indeed, I need to remember to remove these debugging elements before committing.